pyramid/viewderivers.py

```diff
@@ -286,29 +286,27 @@ def _secured_view(view, info):
     authn_policy = info.registry.queryUtility(IAuthenticationPolicy)
     authz_policy = info.registry.queryUtility(IAuthorizationPolicy)
 
+    # no-op on exception-only views without an explicit permission
+    if explicit_val is None and info.exception_only:
```

Comment:
Registering an exception view with `exception_view_config` we can't set permission at all... if `info.exception_only` is set we should just return the view always... unless I am missing a way in that `explicit_val` could still be true.